### PR TITLE
📝 Minor content fixes relating to TA

### PR DESCRIPTION
**Pull request by olyism**
A PR bundling different fixes together.

- [x] Add JUnit button to Get Started
- [x] Minor grammar fix in the _Other CI_ sentence
- [x] Rust collectors should show beta
- [x] Add coming soon for JavaScript collectors
- [x] Change JavaScript collectors slug to `javascript` not `js` for consistency

### CODE REVIEW

File: pages/test_analytics/javascript_collectors.md.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # JavaScript Collectors
```
Comment (ddbeck) [inline pages/test_analytics/javascript_collectors.md.erb:1]: This is my first encounter on a rename with the Buildkite docs. What do we need to do to make sure links to the former URL redirect to the new one?

**Review by ddbeck — COMMENTED** [review]
One question on this. Thanks!

File: pages/test_analytics/javascript_collectors.md.erb (hunk 1 above)
Comment (karensawrey) [reply pages/test_analytics/javascript_collectors.md.erb:1]: Either update all the links manually or set redirects in routes.rb @ddbeck

File: pages/test_analytics/javascript_collectors.md.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-Test Analytics ships with a Jest test collector.
+Test Analytics ships with a Jest test collector, and [more coming soon](https://github.com/buildkite/test-collector-javascript/issues?q=is%3Aissue+is%3Aopen+label%3A%22test+frameworks%22).
```
Comment (karensawrey) [inline pages/test_analytics/javascript_collectors.md.erb:3]:
```suggestion
Test Analytics ships with a Jest test collector, and [more are coming soon](https://github.com/buildkite/test-collector-javascript/issues?q=is%3Aissue+is%3Aopen+label%3A%22test+frameworks%22).
```

**Review by karensawrey — APPROVED** [review]

**Comment by karensawrey** [comment]
Ship it!

File: pages/test_analytics/javascript_collectors.md.erb (hunk 1 above)
Comment (ddbeck) [reply pages/test_analytics/javascript_collectors.md.erb:1]: Oh great, `routes.rb` is where it's at! [Cool URIs don't change](https://www.w3.org/Provider/Style/URI).

**Comment by karensawrey** [comment]
Where are we with this one except for the merge conflict limbo? :D

**Comment by olyism** [comment]
@karensawrey oops, missed the comms. Conflicts resolved and merged 🚀

**Pull request merged**